Nos/Extensions/String+Markdown.swift

```diff
@@ -7,24 +7,41 @@ extension String {
     func extractURLs() -> (String, [URL]) {
         var urls: [URL] = []
         let mutableString = NSMutableString(string: self)
+
+        urls.append(contentsOf: replaceDomains(in: mutableString))
+        urls.append(contentsOf: replaceNIP05s(in: mutableString))
+
+        replaceOccurrences(of: "^\\s*", with: "", in: mutableString)
+        replaceOccurrences(of: "\\s*$", with: "", in: mutableString)
+        replaceOccurrences(of: "\\n{3,}", with: "\n\n", in: mutableString)
+        
+        return (mutableString as String, urls)
+    }
+
+    private func replaceDomains(in mutableString: NSMutableString) -> [URL] {
```

Comment:
Can you add a comment here? Especially I am wondering after reading the function signature: what is it replacing the domains with?